css/style.css

```diff
@@ -0,0 +1,2886 @@
+    font-size: 10px;
+    margin-top: 1px;
+    margin-left: 1px;
+  }
+
+  .top__img {
+    width: 357px;
+    height: 166px;
+    margin-top: 46px;
+    margin-left: 0;
+  }
+
+  .top__wrapper { 
+    margin-left: 8px;
+    margin-right: 0;
+    margin-top: 22px;
+  }
+
+  .top__wrapper-charset {
+    display: flex;
+    align-items: center;
+    margin-left: 0;
+    margin-right: 0;
+    padding-right: 0;
+    background: url(../img/svg/road-mobile.svg) no-repeat;
+    background-position: 202px 2px;
+  }
+
+  .top__wrapper-item {
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+    margin-bottom: 25px;
+    margin-right: 25px;
+    margin-left: 3px;
+  }
+
+  .charset {
+    margin-right: 21px;
+    margin-left: 10px;
+  }
+
+  .charset__item {
+    margin-right: 0;
+    margin-left: 20px;
+  }
+
+  .charset__first {
+    align-items: flex-end;
+    width: 120px;
+    margin: 0 2px 0 0;
+  }
+
+  .charset__speed {
+    width: 38px;
+    height: 28px;
+    margin-left: 24px;
+    margin-bottom: 0;
+    margin-right: 14px;
+    margin-top: 0;
+  }
+
+  .charset__road {
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .charset__first-big {
+    height: 30px;
+    font-weight: 600;
+    font-size: 27px;
+    line-height: 33px;
+    padding-left: 0;
+    padding-bottom: 0;
+    padding-right: 4px;
+  }
+
+  .charset__first-small {
+    font-size: 15px;
+    padding-bottom: 0;
+    padding-right: 21px;
+    margin-right: 27px;
+  }
+
+  .charset__second {
+    font-size: 10px;
+  }
+
+  .button__outline {
+    color: #e51937;
+    margin-right: 0;
+    margin-left: 0;
+    padding-right: 7px;
+    padding-left: 15px;
+    padding-top: 13px;
+    background-position: 0 12px;
+    background-size: 22px 22px;
+  }
+
+  .button__outline-img {
+    margin-left: 5px;
+  }
+
+  .button__red {
+    padding-left: 18px;
+    padding-right: 18px;
+    padding-bottom: 7px;
+    padding-top: 7px;
+    margin-right: 5px;
+    margin-left: 0;
+    margin-top: 0;
+  }
+
+  .button__text {
+    margin-right: 0;
+    padding-right: 0;
+    padding-left: 0;
+    font-size: 13px;
+  }
+
+  .middle__content {
+    max-width: 100%;
+  }
+
+  .middle__info-row {
+    height: 304px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-size: 400px 80px;
+    background-position: 0 50px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-mobile.png) no-repeat;
+    background-position: 0 60px;
+  }
+
+  .bottom__info {
+    margin-top: 296px;
+  }
+    
+  .bottom__description {
+    padding-top: 16px;
+    padding-left: 31px;
+    padding-right: 0;
+  }
+
+  .bottom__stars {
+    display: none;
+  }
+
+}
+
+@media (max-width: 375px) {
+  .header {
+    margin-top: 0;
+    padding-top: 0;
+    padding-bottom: 27px;
+  }
+
+  .header__logo {
+    padding-left: 0;
+  }
+
+  .header__logo-img {
+    padding: 0;
+    width: 138px;
+    height: 42px;
+    margin-left: 2px;
+  }
+
+  .header__menu { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)